examples/cisco/demo_upgrade.pp

```diff
@@ -27,7 +27,7 @@
   # -rwxr-xr-- 1 root root 723M Mar  2 15:25 nxos.7.0.3.I5.1.bin
   # puppetmaster:files:2011>
 
-  node default {
+  node 'certname' {
```

Comment:
What is the reason for this change?

Reply:
It was part of my testing all resources on both modes, and I didn't want this to run - I will revert it in a new PR.